<ui version="4.0" >
 <class>UITemplatesWizard</class>
 <widget class="QDialog" name="UITemplatesWizard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Templates Wizard</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="lInformations" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lLanguage" >
       <property name="text" >
        <string>Language :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbLanguages" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Choose the language in witch you want to pick templates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lType" >
       <property name="text" >
        <string>Type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTypes" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwTemplates" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Choose a template</string>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="textElideMode" >
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbInformations" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Informations</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>5</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <property name="horizontalSpacing" >
       <number>3</number>
      </property>
      <property name="verticalSpacing" >
       <number>3</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lDestination" >
        <property name="text" >
         <string>Destination :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLineEdit" name="leDestination" >
          <property name="toolTip" >
           <string>File Destination</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbDestination" >
          <property name="toolTip" >
           <string>Click to select a folder for your file(s)</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lProject" >
       <property name="text" >
        <string>Project :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pTreeComboBox" name="cbProjects" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Choose the project in witch you want to add file(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lOperators" >
       <property name="text" >
        <string>Operator :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbOperators" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbAddToProject" >
       <property name="toolTip" >
        <string>If checked, file(s) will be add to the current selected project</string>
       </property>
       <property name="text" >
        <string>Add to project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbOpen" >
       <property name="toolTip" >
        <string>If checked the file(s) will be opened at the wizard end</string>
       </property>
       <property name="text" >
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>261</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCreate" >
       <property name="text" >
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pTreeComboBox</class>
   <extends>QComboBox</extends>
   <header>pTreeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>UITemplatesWizard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>153</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel" >
     <x>160</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
